content/1.md

```diff
@@ -1,8 +1,103 @@
 ---
-title: 1 - Windows powertools & Firefox extensions
-description: Ensimmäinen episodi juhuu!
-date: 2021-01-01
+title: '1 – Windows powertools & Firefox extensions 🦾'
+description: |
+  Pyöräytetään uusi podi käyntiin metajaksolla!
+  Mitkä ovat devaajan parhaat tehotyökalut Windowsille?
```

Comment:
Not sure about the word "devaajan" because most apps and extensions were suitable for all power users, not specifically for developers. 🤷‍♂️ Thoughts, @Steellow?

Reply:
I think it's good cus our target audience is developers 👍🏽